maat-court-data-api/src/main/java/gov/uk/courtdata/billing/repository/ApplicantCclfResetRepository.java

```diff
@@ -0,0 +1,28 @@
+package gov.uk.courtdata.billing.repository;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Collections;
+
+@Repository
+@RequiredArgsConstructor
+public class ApplicantCclfResetRepository {
+
+    private static final String RESET_CCLF_SQL = """
+        UPDATE APPLICANTS
```

Comment:
You'll probably run into the same problem I did here, in that you'll need to prefix the table with the `TOGDATA` schema to not run into issues.